fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -744,6 +744,9 @@ std::set<Tag> CommitBatchContext::getWrittenTagsPreResolution() {
 		}
 	}
 
+	toCommit.storeRandomRouterTag();
```

Comment:
This needs to be called only log routers are present?

Reply:
do you have a simulation test where you found this?

Reply:
Actually the above test failure is on "main", I didn't run any tests over this change on this branch.